web/html/src/manager/proxy/container-config.tsx

```diff
@@ -0,0 +1,401 @@
+import * as React from "react";
+
+import SpaRenderer from "core/spa/spa-renderer";
+
+import { AsyncButton } from "components/buttons";
+import { Messages } from "components/messages";
+import { TopPanel } from "components/panels/TopPanel";
+
+enum SSLMode {
+  UseSSL = "use-ssl",
+  CreateSSL = "create-ssl",
+}
+
+type Props = {};
+
+type State = {
+  email: "";
+  sslMode: SSLMode;
+  loading: boolean;
+  maxSquidCacheSize: string;
+  messages: any[];
```

Comment:
This _looks_ like it might be `MessageType[]`, but feel free to correct me if I'm wrong.